go.mod

```diff
@@ -10,12 +10,13 @@ require (
 	github.com/blang/semver v3.5.1+incompatible // indirect
 	github.com/buger/goterm v0.0.0-20181115115552-c206103e1f37
 	github.com/checkpoint-restore/go-criu v0.0.0-20190109184317-bdb7599cd87b
+	github.com/containerd/console v0.0.0-20181022165439-0650fd9eeb50 // indirect
 	github.com/containerd/continuity v0.0.0-20190426062206-aaeac12a7ffc // indirect
 	github.com/containernetworking/cni v0.7.1
 	github.com/containernetworking/plugins v0.8.1
-	github.com/containers/buildah v1.11.2
+	github.com/containers/buildah v1.9.0
```

Comment:
Eek? I think we downgraded pretty far here

Reply:
recent updated commit avoids this, so maybe we're ok for now (tests still failing..)